src/Bicep.LangServer.IntegrationTests/Helpers/IntegrationTestHelper.cs

```diff
@@ -42,7 +42,6 @@ public static async Task<ILanguageClient> StartServerWithClientConnectionAsync(T
             creationOptions = creationOptions with
             {
                 SnippetsProvider = creationOptions.SnippetsProvider ?? SnippetsProvider,
-                NamespaceProvider = creationOptions.NamespaceProvider ?? TestTypeHelper.CreateEmptyProvider(),
```

Comment:
Makes it so the test server uses the same set of type definitions (AzTypes) as the normal bicep invocation.